src/MSAAdvanced.sol

```diff
@@ -342,4 +352,26 @@ contract MSAAdvanced is IMSA, ExecutionHelper, ModuleManager, HookManager {
         (bool success,) = bootstrap.delegatecall(bootstrapCall);
         if (!success) revert();
     }
+
+    function isAccountInitialised() external view virtual returns (bool) {
+        return isAlreadyInitialized();
+    }
+
+    function initAccount(PackedUserOperation memory userOp) internal {
+        if (!isAlreadyInitialized()) {
+            (bytes memory signature, bytes memory initData) =
```

Comment:
does this means the initData is not signed ? 
that is, anyone can replace the initData and re-use original UserOp's signature.

Reply:
Ah this is outdated - didn’t realise this pr is still open

Check out https://github.com/erc7579/erc7579-implementation/pull/35 and then https://github.com/erc7579/erc7579-implementation/pull/38 for the current approach